pkg/agent/core/ngt/handler/grpc/insert_test.go

```diff
@@ -1408,100 +1423,1288 @@ func Test_server_Insert(t *testing.T) {
+		func() test {
+			insertCnt := 100
+			invalidDim := f32VecDim + 1
+			reqs, err := request.GenMultiInsertReq(request.Float, vector.Gaussian, insertCnt, invalidDim, nil)
+			if err != nil {
+				t.Fatal(err)
+			}
+
+			return test{
+				name: "Equivalence Class Testing case 3.3: Fail to StreamInsert 100 vector with all vector with different dimension",
+				args: args{
+					insertReqs: reqs.Requests,
+				},
+				fields: fields{
+					name:              name,
+					ip:                ip,
+					streamConcurrency: streamConcurrency,
+					ngtCfg:            defaultF32SvcCfg,
+				},
+				want: want{
+					errCode: codes.InvalidArgument,
+					rpcResp: func() []*payload.Object_StreamLocation {
+						l := make([]*payload.Object_StreamLocation, insertCnt)
+
+						for i := 0; i < insertCnt; i++ {
+							l[i] = genObjectStreamLoc(codes.InvalidArgument)
+						}
+
+						return l
+					}(),
+				},
+			}
+		}(),
+		func() test {
+			insertCnt := 1
+			reqs, err := request.GenMultiInsertReq(request.Float, vector.Gaussian, insertCnt, intVecDim, nil)
+			if err != nil {
+				t.Fatal(err)
+			}
+
+			return test{
+				name: "Equivalence Class Testing case 4.1: Success to StreamInsert 1 vector",
+				args: args{
+					insertReqs: reqs.Requests,
+				},
+				fields: fields{
+					name:              name,
+					ip:                ip,
+					streamConcurrency: streamConcurrency,
+					ngtCfg:            defaultIntSvcCfg,
+				},
+				want: want{
+					rpcResp: request.GenObjectStreamLocation(insertCnt, name, ip),
+				},
+			}
+		}(),
+		func() test {
+			reqs := []*payload.Insert_Request{
+				{
+					Vector: &payload.Object_Vector{
+						Id:     "uuid-1",
+						Vector: vector.GenSameValueVec(intVecDim, 0),
+					},
+				},
+			}
+
+			return test{
+				name: "Boundary Value Testing case 1.1: Success to StreamInsert with 0 value vector (vector type is uint8)",
+				args: args{
+					insertReqs: reqs,
+				},
+				fields: fields{
+					name:              name,
+					ip:                ip,
+					streamConcurrency: streamConcurrency,
+					ngtCfg:            defaultIntSvcCfg,
+				},
+				want: want{
+					rpcResp: request.GenObjectStreamLocation(1, name, ip),
+				},
+			}
+		}(),
+		func() test {
+			reqs := []*payload.Insert_Request{
+				{
+					Vector: &payload.Object_Vector{
+						Id:     "uuid-1",
+						Vector: vector.GenSameValueVec(f32VecDim, 0),
+					},
+				},
+			}
+
+			return test{
+				name: "Boundary Value Testing case 1.2: Success to StreamInsert with 0 value vector (vector type is float32)",
+				args: args{
+					insertReqs: reqs,
+				},
+				fields: fields{
+					name:              name,
+					ip:                ip,
+					streamConcurrency: streamConcurrency,
+					ngtCfg:            defaultF32SvcCfg,
+				},
+				want: want{
+					rpcResp: request.GenObjectStreamLocation(1, name, ip),
+				},
+			}
+		}(),
+		func() test {
+			reqs := []*payload.Insert_Request{
+				{
+					Vector: &payload.Object_Vector{
+						Id:     "uuid-1",
```

Comment:
You can define `uuid-1` as the default value.
Many test cases use it.

Reply:
fixed :)